operations/commit_queue.go

```diff
@@ -29,6 +29,7 @@ const (
 	commitShaFlag       = "commit-sha"
 	commitMessageFlag   = "commit-message"
 	githubAuthorFlag    = "author"
+	patchAuthorFlag     = "author"
```

Comment:
It seems like the motivation behind this ask is for me for example, to be able to run a patch that you initially created, but have it say "author: Chaya Malik" and not "author: Annie Black". 
Do we know that there is specifically the need to be able to create a patch with a completely third party author? I think we should avoid that if possible, because I'd for example feel a bit weird if patches started showing up on my my patches page that I didn't create.

Reply:
I suppose we could filter this out of the "My patches" page (or allow to toggle the viewing of them), or we could label the patches in the frontend in a way that makes it clear that they were submitted on your behalf.

Reply:
The ask here is because this command will be run by a bot user -- DAG is going to add a button to their tooling that calls this command. Rather than having all new patches created be authored by the bot user, they'd like the author to be the person who hits the button. This is a highly gated permission though, and similar to the flag we already have for the commit queue bot https://github.com/evergreen-ci/evergreen/blob/4afe0f066dd383e47d54cac87858ba7d3d417f11/operations/commit_queue.go#L175. I can open a new ticket to add a Submitter field or something to display also who submitted the patch, but I'd like to avoid making that part of this work.

Reply:
I see, if it's primarily intended for bot users then I'm fine to keep things as is. As for the ticket, I'm ambivalent on whether it would be helpful - I'll defer to your decision

Reply:
If the author is intended to be the user who clicked the button, then I think that makes, as they will expecting to see it in their my patches page. I don't think it's necessary in that case.

Reply:
Actually in the spirit of not creating tickets we don't think we'll do I'll wait to see if this ends up being needed - it hasn't been for the commit queue version of the flag.